pkgs/development/tools/carto/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib
+, buildNpmPackage
+, fetchFromGitHub
+}:
+
+buildNpmPackage rec {
+  pname = "carto";
+  version = "1.2.0";
+
+  src = fetchFromGitHub {
+    owner = "mapbox";
+    repo = "carto";
+    rev = "v${version}";
+    hash = "sha256-TylMgb2EI52uFmVeMJiQltgNCSh6MutFwUjsYC7gfEA=";
+  };
+
+  npmDepsHash = "sha256-8M9hze71bQWhyxcXeI/EOr0SQ+tx8Lb9LfvnGxYYo0A=";
+
+  dontNpmBuild = true;
+
+  meta = {
+    changelog = "https://github.com/mapbox/carto/blob/${src.rev}/CHANGELOG.md";
+    description = "Fast CSS-like map stylesheets";
+    homepage = "https://github.com/mapbox/carto";
+    license = lib.licenses.asl20;
+    mainProgram = "carto";
```

Comment:
Both the package and the binary it provides are called "carto", right? In this case you should remove `mainProgram` as it is unnecessary.
```suggestion
```

Reply:
Not anymore: https://github.com/NixOS/nixpkgs/pull/246386

Reply:
Thanks, I didn't know.